_posts/2020-05-13-how-we-do-pull-requests.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "How we do Pull Requests"
+author: michaeltlombardi
+categories:
+  - team
+  - basics
+tags:
+  - git
+  - github
+  - PRs
+  - pull requests
+  - code review
+---
+
+[This was originally posted as [a twitter thread](https://twitter.com/TrebuchetOps/status/1257781807870816257) by [Michael](https://twitter.com/TrebuchetOps). This version edited by [me](https://github.com/DavidS). I've tried to stay faithful to the original tone while unraveling twitter lingo. all errors are my own.]
+
+[ReinH](https://twitter.com/ReinH)'s [thread on PRs](https://twitter.com/ReinH/status/1257773735815213057) is a good thread and while I can't speak to how anyone else uses PRs, I can talk about how my team does.
+
+Background: we're a team of about 10 people (I'm on [paternity leave](https://twitter.com/TrebuchetOps/status/1256985345830400002) as of Sunday, so not poking my head into slack to double check my math) with about 75 "supported" project repos and another 100+ "unsupported" - including puppet modules, Ruby gems, PowerShell modules, and more.
+We process (submit, merge, or close) over 350 PRs a month from our own team, from community folx, from other people at Puppet.
+
+So we rely on PRs a lot, but for what?
+
+Collaboration and safety, mostly!
+
+We very often throw up early WIP (Work In Progress) PRs and ping them around the team, solicit and provide feedback, pair or help each other, using the PR as our main point of syncing context on the code.
+We supplement this with other tools, of course!
+
+But my team is also spread across two continents and three timezones, (I'm the only USAian on my team, everyone else is in the Belfast or Timișoara), so async-first work is the only way we can really get things done.
+PRs support this!
+We can comment, posit questions, make suggestions, add context, link resources, etc all on the PR, so you can push your WIP and return to it tomorrow (or earlier, most of the time!)
```

Comment:
```suggestion
We can comment, post questions, make suggestions, add context, link resources, etc all on the PR, so you can push your WIP and return to it tomorrow (or earlier, most of the time!)
```

Reply:
https://dictionary.cambridge.org/dictionary/english/posit

Reply:
That said - might still be an argument to simplify the flourish here for easier reading.